### PR TITLE
Initial doc commit

### PR DESCRIPTION
**Pull request by e1mo**

### CODE REVIEW

File: doc/admin/installation/index.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+When visiting http://[::1]:1234/ we will be greeted by hello world page (for now).
+
+.. todo:: Systemd service
+
+.. todo:: nginx configuration with ssl
+
+.. _gunicorn: https://gunicorn.org/
+.. _PostgreSQL: https://www.postgresql.org/
+.. _virtual environment: https://docs.python.org/3/library/venv.html
```
Comment (margau) [inline doc/admin/installation/index.rst:92]: Maybe add a paragraph about future deployment methods and "help wanted" over here?

File: doc/conf.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        'chaosinventory': ('https://chaosinventory.readthedocs.io/en/latest/', None), # noqa
+        'django': ('http://docs.djangoproject.com/en/stable/', 'http://docs.djangoproject.com/en/stable/_objects/'), # noqa
+    }
+
+# -- Options for todo extension ----------------------------------------------
+
+# If true, `todo` and `todoList` produce output, else they produce nothing.
+todo_include_todos = True
+
+manpages_url = 'https://man.archlinux.org/man/{page}.{section}'
```
Comment (margau) [inline doc/conf.py:84]: What does this do over here?

**Review by margau — APPROVED** [review]

**Comment by margau** [comment]
Please not the non-guranteed migrations yet, as discussed today

File: doc/conf.py (hunk 2 above)
Comment (e1mo) [reply doc/conf.py:84]: See https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-manpage (and https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-manpages_url )

**Review by em0lar — APPROVED** [review]

**Comment by em0lar** [comment]
closed with bbe3a9111df965337e72c5b157256388fb11054a

**Pull request closed without merging**